src/mina-signer/test-vectors/legacySignatures.ts

```diff
@@ -0,0 +1,206 @@
+import { DelegationJson, PaymentJson } from '../sign-legacy.js';
```

Comment:
Are any of these imported from https://github.com/o1-labs/proof-systems/blob/226de4aeb11b8814327ab832e4fccdce5585f473/signer/tests/signer.rs#L63
?

Reply:
no - they come from a test that existed on the OCaml side, which was used to check consistency of the OCaml and JS versions of mina-signer: https://github.com/MinaProtocol/mina/blob/develop/scripts/compare_test_signatures.sh

I just ran the script that generated those test signatures and copied the output in this file